### PR TITLE
Fix #15895: make navigation icons in a correct position

### LINKED ISSUE #15895: navigation icons are not positioned correctly (metro theme)
### Describe the bug

PHPMyAdmin logo is missing and navigation icons are not positioned correctly in the left navigation panel (in the metro theme)

### To Reproduce

1. Go to 'home'
2. Select 'Metro' theme
4. See the left navigation panel.

### Screenshots

logomissing0

### Expected behavior

There should be the logo of PHPMyAdmin and all navigation icons must be properly positioned and unnecessary margin/padding should be removed.

like: 
logomissing1

### PR DESCRIPTION
**Pull request by MouadBH**
Signed-off-by: Mouad Boulahdoud <mouad123b@gmail.com>

### Description

fix #15895

### Fixes
Before Fix:
image
After Fix:

image

### CODE REVIEW

**Comment by MouadBH** [comment]
@williamdes  Please review and let me know if any changes are required. :)

File: .gitignore [hunk 1]
```diff
@@ -57,3 +57,4 @@ composer.lock
 # NPM
 /node_modules/
 phpstan.neon
+.yarn
```
Comment (williamdes) [inline .gitignore]: This change could be removed

File: themes/metro/scss/_navigation.scss [hunk 2]
```diff
@@ -83,11 +83,14 @@
       text-transform: uppercase;
       margin-#{$left}: 5px;
       content: "phpMyAdmin";
+      display: none;
```
Comment (williamdes) [inline themes/metro/scss/_navigation.scss]: I am not sure this is okay

**Review by williamdes — CHANGES_REQUESTED** [review]
I added some comments, I think you can remove the new logo file and the gitignore change and then amend your commit
Also you maybe need to change your git settings for your name or just change the signoff line before next push (--force) :)

File: themes/metro/scss/_navigation.scss (hunk 2 above)
Comment (MouadBH) [reply themes/metro/scss/_navigation.scss]: so we can delete the old logo

Comment (williamdes) [reply themes/metro/scss/_navigation.scss]: I will need to test it at the end of the week :)

File: .gitignore (hunk 1 above)
Comment (MouadBH) [reply .gitignore]: @williamdes I push a new commit to fix what you tell me.

**Comment by AbdullahiAbdulkabir** [comment]
I found out that the navigation bar was okay in the cloned project and that the ::after is affecting the logo from displaying and also the display of the image was set to none. 
After making the image to be visible there was still a bit of CSS adjustment that I made.

File: .gitignore [hunk 3]
```diff
@@ -56,4 +56,4 @@ composer.lock
 /vendor/
 # NPM
 /node_modules/
-phpstan.neon
+phpstan.neon
```
Comment (mauriciofauth) [inline .gitignore]: Please revert this change.

**Review by MauricioFauth — CHANGES_REQUESTED** [review]
LGTM

**Review by williamdes — APPROVED** [review]

**Comment by williamdes** [comment]
I made the requested change for you.
Thank you for your contribution

**Pull request merged**